wiring/src/spark_wiring_ble.cpp

```diff
@@ -1587,6 +1526,31 @@ bool BlePeerDevice::getCharacteristicByUUID(BleCharacteristic& characteristic, c
     return false;
 }
 
+bool BlePeerDevice::getCharacteristicByDescription(const BleService& service, BleCharacteristic& characteristic, const char* desc) const {
+    CHECK_TRUE(desc, false);
+    for (auto& existChar : impl()->characteristics()) {
+        if (existChar.impl()->svcUUID() == service.UUID() && !strcmp(existChar.description().c_str(), desc)) {
```

Comment:
This also checks for matching UUIDs, so it will be a problem if GATT server has multiple services with the same UUID. Can we instead check for handles being in range?